app/models/concerns/conference_override.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+module ConferenceOverride
+  extend ActiveSupport::Concern
+
+  included do
+    def self.filtered_by_hashtag(hashtag)
```

Comment:
Can we create a simple spec for this model? and test that it works with many combination of the hastag word (in the middle, at the beginning, with the `#` sign, etc).

Btw, you might want to simplify the query using (and leave out the `#` sign):

```ruby
where("CONCAT(' ', hashtag, ' ') ILIKE ?",   "% #{hashtag} %")
```